### ISSUE TITLE
[Feature] Add support for specific pages instead of entire site

### ISSUE THREAD

**Issue by wackyblackie** [ankit/stylebot#657]
### Is your feature request related to a problem? Please describe.
I am quite fond of Stylebot, but one issue I've always had is the inability to style just one page, not the entire site.

### Describe the solution you'd like
I would like an option to select a specific page _**or**_ the entire site. This is could, perhaps, be a toggle int

### Additional context
I know this works by going to `Options > Styles > site.domain > Edit` and renaming it to `site.domain/only-styled-page`, but it still applies the styles for the main-page as well (unless disabled directly by me).

**Comment by ankit** [comment]
@wackyblackie makes sense, having the ability to adjust the specificity/url in the stylebot editor would be helpful. 
I think the issue with renaming it to something specific and it still getting applied to the main page should be fixed as part of another fix
